src/Microsoft.DotNet.Interactive.HttpRequest.Tests/ParserTests.cs

```diff
@@ -71,8 +71,26 @@ public void multiple_whitespaces_are_treated_as_a_single_token()
             .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
             .MethodNode.ChildTokens.First().TextWithTrivia.Should().Be("     ");
         }
+
+        [Fact]
+        public void multiple_newlines_are_parsed_into_different_tokens()
+        {
+            var result = HttpRequestParser.Parse("\n\n\r\n\n");
+            result.SyntaxTree.RootNode
+                  .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
+                  .MethodNode.ChildTokens.Count().Should().Be(4);
+        }
+
+        [Fact]
+        public void multiple_punctuations_are_parsed_into_different_tokens()
+        {
+            var result = HttpRequestParser.Parse(".!?.");
+            result.SyntaxTree.RootNode
+                  .ChildNodes.Should().ContainSingle<HttpRequestNode>().Which
+                  .UrlNode.ChildTokens.Count().Should().Be(4);
```

Comment:
This test should ideally check that there are 4 tokens and that their `token.Kind is TokenKind.Punctuation`.

Reply:
We have a few FluentAssertions tricks for this. Easier to pair on it.

Reply:
Resolving since this is fixed.